husky_gazebo/launch/controller_utils.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+
+  <!-- Robot state publisher -->
+  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
+    <param name="publish_frequency" type="double" value="50.0" />
+    <param name="tf_prefix" type="string" value="" />
+  </node>
+
+ <!-- Joint state publisher -->
+  <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher"/>
```

Comment:
In addition, running view_model calls description.launch again (without the ability to pass in ur5_enabled.. so the ur5 can never spawn). 

I'm not sure of the repercussions of these changes.. but:
view_model will have to have a parameter passed in (ur5_enabled) or view_robot will have to spawn its own JSP (which I think might crash the JSP that the robots internals spawn so this is why it was architected this way)